raw_data/preprocessed/africa/seychelles_english.txt

```diff
@@ -0,0 +1,91 @@
+
+(30) [1] Any person who commits an offence under section 16[2] or section 17[5] or section 22[2] or section 22[3] is liable on conviction, to imprisonment for one year and a fine of R50.000 and, if the offence is continued after conviction, is liable to a further fine of R5.000 for each day during which the offence is so continued. [2] A person who commits an offence under sections 7[4][a] or section 7[5] or section 8[2] or section 8[3] or section 9[2] or section 11[4] or section 12[3] or section 12[6] section 12[7] or section 12[8][a] or section 12[8][b] or section 14[1] or section 15[1] or section 15[14] is liable, on conviction, to imprisonment for 6 years and a fine of R250.000 and, if the offence is continued after conviction, is liable to a further fine of R5.000 for each day during which the offence is so continued. [3] Where an offence specified in subsection [2] is continued beyond a period of one year after conviction by the person so convicted the person is guilty of an offence and liable on conviction to imprisonment for not less than 2 years and not exceeding 7 years.
+
+(31) Any person who: [a] destroys , pulls down, removes, injures or defaces any pillar, post or stake fixed by the Authority or any notice or other matter put up, inscribed or placed by or under the order of the Authority; or [b] obstructs any person acting under the orders or directions of the Authority from exercising the powers and performing the functions under this Act; or [c] damages any works or property belonging to the Authority, or [d] fails to furnish to any officer or other employee of the Authority a report or any information required by the officer or employee for the purpose of this Act, or [e] fails to intimate the occurrence of any accident or other unforeseen act to the Authority, or [f] in giving a report or any information which the person is required to give under this Act, makes a statement which is false or misleading in any material particular, is guilty of an offence and is liable on conviction to imprisonment for 6 months and a fine of R10.000.
```

Comment:
Please replace:
* `destroys , pulls` with `destroys, pulls`;
* `Authority, or [d]` with `Authority; or [d]`;
* `Act, or [e]` with `Act; or [e]`;
* `Authority, or [f]` with `Authority; or [f]`;
* `particular, is` with `particular; is`; and
* `10.000` with `10,000`.